pandas/tests/resample/test_resample_api.py

```diff
@@ -233,6 +233,7 @@ def test_apply_without_aggregation():
         assert_series_equal(result, test_series)
 
 
+@pytest.mark.skipif(not PY36, reason='Insertion order for Python>=3.6')
```

Comment:
is this needed?

Reply:
How does the changes in this PR affect this case?